pcp-htop.c

```diff
@@ -14,13 +14,14 @@ in the source distribution for its full text.
 #include "Platform.h"
 
 
+const char* program = "pcp-htop";
```

Comment:
Maybe try to get this name from `autotools` too?

Reply:
Hmm, not seeing an obvious way to do that after a quick look, so I may leave this idea if that's OK?

Reply:
Sure, just was a suggestion in case it's available. Would have been a bit more consistent with `htop.c` where it's fetched from `PACKAGE_NAME` directly.